src/containers/Editor/BottomBar.tsx

```diff
@@ -273,6 +279,13 @@ export const BottomBar = () => {
             Transform
           </StyledBottomBarItem>
         )}
+
+        {isNotExploitableJson(contents) && (
+          <StyledBottomBarItem>
+            <AiOutlineSafetyCertificate />
+            JSON is not exploitable
+          </StyledBottomBarItem>
+        )}
```

Comment:
It's better to show the warning when it's **exploitable** instead of **not exploitable**.

Reply:
Also we could use "Exploitable" instead of "JSON is exploitable" so it will consume less space.